### PR TITLE
add note about limitation of throughput in port mirroring cases

### PR DESCRIPTION
**Pull request by jklare**
Signed-off-by: Jan Klare <jan.klare@bisdn.de>

### CODE REVIEW

File: platform_configuration/port_mirroring.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ BISDN Linux supports the configuration of mirror ports. Mirrored ports replicate
 * `OFDPA_MIRROR_PORT_TYPE_EGRESS`: Only egress traffic is mirrored
 * `OFDPA_MIRROR_PORT_TYPE_INGRESS_EGRESS`: Both ingress and egress traffic are mirrored
 
+Please note that when mirroring ports with different throughputs (e.g. a 10G
```
Comment (rr-santos) [inline platform_configuration/port_mirroring.md:15]:
```suggestion
Please note that when mirroring ports with different maximum link speeds (e.g. a 10G
```

File: platform_configuration/port_mirroring.md [hunk 2]
```diff
@@ -12,6 +12,11 @@ BISDN Linux supports the configuration of mirror ports. Mirrored ports replicate
...
+port mirrored to a 1G port), the highest common throughput (1G for the before
```
Comment (rr-santos) [inline platform_configuration/port_mirroring.md:16]:
```suggestion
port mirrored to a 1G port), the highest common link speed (1G for the
```

File: platform_configuration/port_mirroring.md [hunk 3]
```diff
@@ -12,6 +12,11 @@ BISDN Linux supports the configuration of mirror ports. Mirrored ports replicate
...
+mentioned example) will be used for both ports.
```
Comment (rr-santos) [inline platform_configuration/port_mirroring.md:17]:
```suggestion
aforementioned example) will be used for both ports.
```

**Review by rr-santos — CHANGES_REQUESTED** [review]

**Comment by jklare** [comment]
I’ll squash the commits and push again (otherwise the DCO check will be sad)

**Pull request merged**